kalliope/core/OrderAnalyser.py

```diff
@@ -56,6 +58,9 @@ def get_matching_synapse(cls, order, brain=None):
         # create a list of MatchedSynapse from the tuple list
         list_synapse_to_process = cls.get_list_synapses_to_process(list_match_synapse)
 
+        # save in kalliope memory the last stt order
+        Cortex.save("kalliope_last_order", order)
```

Comment:
this one is not needed I think as it is already present [here](https://github.com/kalliope-project/kalliope/blob/383ab40f00bd2f52f48642c87b23c33ef0584909/kalliope/signals/order/order.py#L171)